README.md

```diff
@@ -39,7 +35,7 @@ Mobly tests could run on the following platforms:
 
 ## System dependencies
   - adb (1.0.36+ recommended)
```

Comment:
can probably change this to `1.0.40+` as well

Reply:
Thank you for this comment.